src/libraries/System.Net.Primitives/tests/UnitTests/CookieContainerTest.cs

```diff
@@ -125,11 +125,11 @@ public static IEnumerable<object[]> SetCookiesData()
             yield return new object[]
             {
                 u,
-                "name98=value98; path=/; domain=.uri.com; expires=Wed, 09 Jun 2021 10:18:14 GMT, name99=value99",
+                "name98=value98; path=/; domain=.uri.com; expires=Wed, 09 Jun 2050 10:18:14 GMT, name99=value99",
```

Comment:
I'll be really excited if this test is still running in 29 years and fails then.